releasing_guide.md

```diff
@@ -5,8 +5,8 @@ Cut a Release
 Create a PR to the `RELEASE_next_patch` branch and go through the following steps:
 
 **Preparation**
-- Update and check changelog in `CHANGES.rst`
 - Bump version in `hyperspy/Release.py`
+- Update and check changelog in `CHANGES.rst` (run `towncrier build`) 
```

Comment:
Even if the target audience of the releasing guide is not first time pull requester ;), it doesn't hurt mentioning the draft option, particularly since it can save us having to look up for the syntax!